lib/linter-julia.coffee

```diff
@@ -23,9 +23,9 @@ doSomeMagic = (data,textEditor) ->
   lines = data.split("\n")
   for line in lines
     try
-      splittedline = line.split(":")
-      numbers = splittedline[1].split(" ")
-      if numbers[1] in ignore
+      splittedline = line.replace(filePath+":","").split(":")
+      numbers = splittedline[0].split(" ")
+      if numbers.length < 2 || numbers[1] in ignore
```

Comment:
Can you explain, why this is needed: `numbers.length < 2 ||`? I'm not against it, just curious.